api/src/main/java/org/ccci/idm/user/DefaultUserManager.java

```diff
@@ -34,12 +36,20 @@
     private static final EmailValidator VALIDATOR_EMAIL = EmailValidator.getInstance();
 
     @NotNull
+    @Autowired(required = false)
```

Comment:
That is, could you not use `@Nullable` as you do here https://github.com/GlobalTechnology/idm-user-management/blob/master/ldaptive/src/main/java/org/ccci/idm/user/ldaptive/dao/LdaptiveUserDao.java#L93 ?